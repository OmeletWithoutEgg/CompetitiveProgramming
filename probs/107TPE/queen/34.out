-215617.19 -35287421.65
