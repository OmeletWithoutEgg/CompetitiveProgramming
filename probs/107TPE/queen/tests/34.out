50337.77 61553863.72
